File: COT_TOT_PYTHON/input_path/spacy_input_path/bug7.py
"""
Buggy NON-MODULAR snippet from spacy, bug ID 7.
"""
def buggy_function(rs, data, freq=None):
Last tested with: v2.1.0
        get_sort_key = lambda span: (span.end - span.start, span.start)
                seen_tokens.update(range(span.start, span.end))
        get_sort_key = lambda span: (span.end - span.start, span.start)
    return rs

File: COT_TOT_C/agent4_results_tot/ansible_agent4_output_code/bug7.c
#include <stdio.h>
int buggy_function(int rs, char *data, int freq) {
    char *to_remove[]={"a","b","c"};
    int i;
    for(i=0;i<3;i++)
    {
        printf("no %s\n",to_remove[i]);
    }
    return rs;
}